examples/energy-management-app/energy-management-common/include/FakeReadings.h

```diff
@@ -0,0 +1,118 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app-common/zap-generated/cluster-objects.h>
+#include <lib/core/DataModelTypes.h>
+#include <system/SystemLayer.h>
+
+#define MAX_SLOTS 10
```

Comment:
prefer C++ style constexpr constants (which are typesafe) to C-style macros.

Reply:
Where is this used? Without a namespace a name of `MAX_SLOTS` is too generic.

Reply:
MAX_SLOTS is no longer being used in FakeReadings.h so remove. I have replaced #defines with constexp elsewhere.